j5basic/Module.py

```diff
@@ -103,7 +103,11 @@ def get_all_distinct_mro_targets(obj, functionname):
     for t in reversed(obj.__mro__):
         base_hook_fn = getattr(t, functionname, None)
         if base_hook_fn:
-            t_f = base_hook_fn.__func__
+            t_f = getattr(base_hook_fn, '__func__', None)
+            if t_f is None:
```

Comment:
I don't understand, won't `t_f` always be `None` on Python 3?

Reply:
Nope, not for the sqlalchemy recordclasses we're using this method on it seems. Normally yes though, for any other sort of class this would be the case.